### ISSUE TITLE
Ambiguous level feedback

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#92]
The file
`RobotFramework_TestsuitesManagement\Keywords\CSetup.py`
contains the following lines:
```
logger.warn(f"Running with configuration level 1! \nSelected configuration file '{RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sTestCfgFile}'")
logger.info('Running with configuration level 1')
logger.info('Running with configuration level 2')
logger.info('Running with configuration level 3')
logger.warn(f"Running with configuration level 4! \nSelected configuration file '{RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sTestCfgFile}'")
```

The first line prints level 1 as warning, the second line as info. Why do you make a difference here? The feedback of a certain level should be unique. I would prefer to handle level 1 as info.

I also assume that the first line can be removed because a feedback about level 1 is also given some lines below. Redundancy?

In this line
`RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.rConfigFiles.sLevel2 = True`
the type prefix `s` of `sLevel2` indicates a string, but you assign a boolean value. Therefore the type prefix should be adapted. Also other levels.

**Comment by namsonx** [comment]
Hello Holger,

Thank you for your finding!
Firstly, I don't remember exactly in which circumstance we had put the warning for level1 configuration here, but it seems to announce users that robot is running with specific configuration file instead of choosing configuration file via variant name.

For now, our documentation already mentions very clear about 4 levels configuration, so I agree that this warning message is no longer necessary.

Thank you,
Son

**Comment by namsonx** [comment]
Hello Holger,

I created pull-request #101  for this enhancement.

Thank you,
Son

**Comment by test-fullautomation** [comment]
solved and already merged with #101
